src/dart_diagnostic_provider.ts

```diff
@@ -17,7 +17,7 @@ export class DartDiagnosticProvider {
 
 	private handleErrors(notification: as.AnalysisErrorsNotification) {
 		let errors = notification.errors;
-		if (!getConfig('showTodos'))
+		if (!getConfig<boolean>('showTodos'))
```

Comment:
> it applies immediately

ah, that's very nice. the analysis server output can be very useful but high volume. It's great to be able to toggle it live.